src/js/components/Menu/stories/typescript/Children.tsx

```diff
@@ -13,15 +13,15 @@ const MenuWithChildren = (props) => (
     {...props}
```

Comment:
In the HPE theme for this story the disabled menu looked the same as the regular menu

Reply:
I believe it is because that story uses `plain` in the props we can take this out ?

Reply:
Yeah I think we could take out plain from this story